src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (x) => {
+    return (y) => {
+      counter++;
+
+      if (counter <= 3 || counter % 2 !== 0) {
+        return x + y;
+      } else if (counter % 2 === 0) {
```

Comment:
you do not need this condition `else if (counter % 2 ===)` 
you can write just `return 'Bzzz... Error!';` from new line